lib/automaticupgrades/version/proxy.go

```diff
@@ -28,11 +28,15 @@ import (
 	"github.com/gravitational/teleport/lib/automaticupgrades/constants"
 )
 
+type Finder interface {
+	Find() (*webclient.PingResponse, error)
```

Comment:
Not relevant to this PR, but this method should accept a context for proper cancellation of IO

Reply:
I agree, but it's not that bad because there's a hidden context & deadline in the web client itself, so it will get cancelled if we stop the controller or if the call hangs.